helix-term/src/application.rs

```diff
@@ -78,20 +85,30 @@ impl Application {
         compositor.push(editor_view);
 
         if !args.files.is_empty() {
-            let first = &args.files[0]; // we know it's not empty
+            let first = &args.files[0].0; // we know it's not empty
             if first.is_dir() {
                 editor.new_file(Action::VerticalSplit);
                 compositor.push(Box::new(ui::file_picker(first.clone())));
             } else {
                 let nr_of_files = args.files.len();
                 editor.open(first.to_path_buf(), Action::VerticalSplit)?;
-                for file in args.files {
+                for (file, row, col) in args.files {
                     if file.is_dir() {
                         return Err(anyhow::anyhow!(
                             "expected a path to file, found a directory. (to open a directory pass it as first argument)"
                         ));
                     } else {
-                        editor.open(file.to_path_buf(), Action::Load)?;
+                        editor.open(file, Action::Load)?;
+                        // let docid = editor.open(file, Action::Load)?;
+                        // editor.document_mut(docid).unwrap().set_selection();
+                        // FIXME: current is incorrect since it will only work for current
+                        // document, other files loaded in background will result in panic
+                        let (view, doc) = current!(editor);
```

Comment:
Not sure how to fix this. It should get the correct view that is not focused.

Reply:
You should do nothing here since the file is loaded in the background and not assigned to a view.

If you want to set a location for the current split, you can push a selection onto the doc `doc.selections.insert(view.id, ...)`

Reply:
> You should do nothing here since the file is loaded in the background and not assigned to a view.

But the user did specify the file position, so I don't think I should do nothing here.